csrc/fused_moe/monomoe/tools/tune_monomoe.py

```
@@ -0,0 +1,453 @@
+    config that failed the accuracy gate (perf is skipped on reject)."""
+    import torch
+
+    # Imported here so Ray has pinned the actor's GPU before torch init.
+    from flashinfer.fused_moe import mono_moe, alloc_scratchpad
+    from flashinfer.testing import bench_gpu_time
+
+    E, N, K = entry["E"], entry["N"], entry["K"]
+    dev = torch.device("cuda")
+
+    # Weights + inputs are shape-only (constant across configs); build once.
+    w13_fp8, s13, w2_fp8, s2, w13_f, w2_f = H.make_weights(dev, E, N, K, seed=42)
```

Ruff (0.15.21)

[warning] Unpacked variable `w13_f` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)